tests/unit/handler.unitTest.ts

```diff
@@ -26,7 +26,7 @@ describe("The lambda function handler", () => {
           queryStringParameters: null
         };
 
-        const ctx: any = mockContext(opts);
+        let ctx: any = mockContext(opts);
```

Comment:
Could put these into beforeEach and afterEach steps, and reduce the repetition. The ctx.succeed(result) isn't really necessary anyway.

Reply:
I think these were done by Arup some time ago when he suggested that we should follow his example from defects

Reply:
I think it was one of his less successful experiments to resolve what turned out to be the  other issue you fixed. I won't be hard nosed about this one, but worth noting for future.